examples/kmeans/kmeans.py

```diff
@@ -0,0 +1,136 @@
+import argparse
+import contextlib
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+import six
+import time
+
+import numpy as np
+
+import cupy
+
+
+@contextlib.contextmanager
+def timer(message):
+    cupy.cuda.Stream.null.synchronize()
+    start = time.time()
+    yield
+    cupy.cuda.Stream.null.synchronize()
+    end = time.time()
+    print('%s:  %f sec' % (message, end - start))
+
+
+def use_custom_kernel(X, n_clusters, max_iter, elem):
```

Comment:
Why did you use this name? My comment suggested to replace `elem` with `use_custom_kernel`.